.travis.yml

```diff
@@ -31,6 +31,7 @@ install:
   - pip install --upgrade pip
   - pip install .
   - pip install -r dev-requirements.txt
+  - mlflow sagemaker build-and-push-container --no-push --mlflow-home ./
```

Comment:
Is this a requirement for tests or is this meant to be a test? Maybe it should go into the `tox` files so people get it when testing locally too? Or if it's meant to be a test, then we can add it as a test case.

Reply:
Good point. It is a requirement for tests. Previously I was building the container in sagemaker test but now I want to test other models in sagemaker (e.g. testing spark model revealed missing java) container and I don't want to be rebuilding the container all the time as it takes time. 

Adding it to tox files makes sense, but I also need to see local mlflow-project. I'll look into it.